content/blog/2017/2017-01-12-scm-api-2.0-release.adoc

```diff
@@ -0,0 +1,230 @@
+---
+:layout: post
+:title: SCM API turns 2.0 and what that means for you
+:tags:
+- development
+- plugins
+:author: stephenc
+---
+
+We've made some great improvements to a number of popular plugins including:
+
+* GitHub Branch Source
+* BitBucket branch source
+* Git
+* Mercurial
+* Branch API
+* SCM API
+* Pipeline Multibranch
+* GitHub Organization Folders
+
+There are some gotcha's that Jenkins administrators will need to be aware of.
+
+We want to give you the whole story, but the <<tldr,TL;DR>> is this
+
+TIP: When updating your plugins, please be sure to upgrade the GitHub Branch Source and/or Bitbucket branch source plugins.
+
+WARNING: Doing partial upgrades is bad news!
+
+If you don't care about the hows and whys, you can just skip down to <<tldr,this section>> but if you are curious... here we go!
+
+== The back-story
+
+Way back in September 2013 we announced the
+link:https://jenkins.io/blog/2013/09/23/literate-builds-wtf/[literate plugin],
+as an experimental new way of modeling branch development in Jenkins.
+
+When you are performing an experiment, the recommendation is to do just enough work to let you perform the test.
+However, the culture in Jenkins is to always try and produce reusable components that others can use in ways you have not anticipated.
+
+So when releasing the initial version of the
+link:https://wiki.jenkins-ci.org/display/JENKINS/Literate+Plugin[literate plugin]
+we also released some reusable components upon which the multi-branch functionality was built.
```

Comment:
s/multi-branch/Pipeline Multibranch and Org Folder/

We introduced those plugins above and that is what people are using and where we want to lead them.